npm-shrinkwrap.json

```diff
@@ -389,7 +389,7 @@
           "dependencies": {
             "nan": {
               "version": "1.3.0",
-              "from": "nan@1.3.0",
+              "from": "https://registry.npmjs.org/nan/-/nan-1.3.0.tgz",
```

Comment:
It seems an internal change to reference the external module better int the repository. But the versions are the same, so I think it's OK.